dev/tools/gen_defaults/lib/tabs_template.dart

```diff
@@ -71,7 +74,7 @@ class _${blockName}PrimaryDefaultsM3 extends TabBarTheme {
   InteractiveInkFeatureFactory? get splashFactory => Theme.of(context).splashFactory;
 
   @override
-  TabAlignment? get tabAlignment => isScrollable ? TabAlignment.start : TabAlignment.fill;
+  TabAlignment? get tabAlignment => isScrollable ? TabAlignment.startOffset : TabAlignment.fill;
```

Comment:
Use the newly added tab alignment to match the material specs for the scrollable tab bar. 


![Screenshot 2023-05-05 at 18 54 00](https://user-images.githubusercontent.com/48603081/236507223-9f4ef4f2-1d8f-48ae-9676-b93cdc354808.png)

Reply:
I think we should run some extra testing on this to ensure it isn't reverted later. 👍